CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ the detailed section referring to by linking pull requests or issues.
 
 - Implemented markdown for asset descriptions
 - Implemented list view for Broker UI
+- Asset Metadata is now editable
```

Comment:
I probably missed some things as some of the longer form files and their interplay were complex to understand :D

Reply:
it was quite a bit of spaghetti code connecting the dialogs